Heroes Handbook, Main File.txt

```diff
@@ -3517,7 +3517,7 @@ For example, if you are a 5th-level paladin, you have four 1st-level and two 2nd
 You can change your list of prepared spells when you finish a long rest. Preparing a new list of paladin spells requires time spent in prayer and meditation: at least 1 minute per spell level for each spell on your list.
 
 #### Spellcasting Ability
-Charisma is your spellcasting ability for your paladin spells, since their power derives from the strength of your convic-tions. You use your Charisma whenever a spell refers to your spellcasting ability. In addition, you use your Charisma modifier when setting the saving throw DC for a paladin spell you cast and when making an attack roll with one.
+Charisma is your spellcasting ability for your paladin spells, since their power derives from the strength of your convictions. You use your Charisma whenever a spell refers to your spellcasting ability. In addition, you use your Charisma modifier when setting the saving throw DC for a paladin spell you cast and when making an attack roll with one.
```

Comment:
```suggestion
Charisma is your spellcasting ability for your paladin spells, since their power derives from the strength of your convic-tions. You use your Charisma whenever a spell refers to your spellcasting ability. In addition, you use your Charisma modifier when setting the saving throw DC for a paladin spell you cast and when making an attack roll with one.
```